bin/sanctuary-benchmark

```diff
@@ -2,74 +2,73 @@
 
 'use strict';
 
-var fs = require ('fs');
+const fs = require ('fs');
 
-var path = require ('path');
+const path = require ('path');
 
 
-var dir = path.resolve (process.cwd (), 'bench');
+const dir = path.resolve (process.cwd (), 'bench');
 
-var help = 'Create a bench directory in this project to get started\n';
+let help = `Create a bench directory in this project to get started
+`;
```

Comment:
I'm equally happy to stick with `\n` in this case. Do you have a preference, Aldwin?

Reply:
I prefer the `\n`. The new version looks like a mistake.